libs/public-pages/src/utils/format-publication-date.ts

```diff
@@ -0,0 +1,17 @@
+export function formatPublicationDate(dateString: string, locale: string): string {
```

Comment:
I wonder the file name and the method name should be something more generic. This can be used for other dates too rather than just publication date